packages/agreement-process/test/activateAgreement.test.ts

```diff
@@ -90,12 +91,21 @@ describe("activate agreement", () => {
     id: generateId(),
   };
 
-  const mockSelfcareUserResponseWithMissingInfo: UserResponse = {
-    ...mockSelfcareUserResponse,
-    name: randomArrayItem([mockSelfcareUserResponse.name, undefined]),
-    surname: randomArrayItem([mockSelfcareUserResponse.surname, undefined]),
-    taxCode: randomArrayItem([mockSelfcareUserResponse.taxCode, undefined]),
-  };
```

Comment:
This could result in a case where no info was missing (name, surname, and taxCode all defined), leading to the expected `agreementMissingUserInfo` error not to happen: https://github.com/pagopa/interop-be-monorepo/actions/runs/9510756215/job/26215757075?pr=646